ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/PostCommitValidation.scala

```diff
@@ -73,7 +75,7 @@ private[dao] object PostCommitValidation {
       * 2. refer exclusively to contracts with a ledger effective time smaller than or equal to the transaction's?
       */
     private def validateCausalMonotonicity(
-        transaction: Transaction,
+        transaction: Tx.Transaction,
```

Comment:
Why not propagate `CommittedTransaction` into here as well?

Reply:
done.